src/main/java/com/example/service/impl/BookServiceImpl.java

```diff
@@ -13,12 +17,23 @@ public class BookServiceImpl implements BookService {
     private final BookRepository bookRepository;
 
     @Override
-    public Book save(Book bookEntity) {
-        return bookRepository.save(bookEntity);
+    public BookDto save(CreateBookRequestDto bookRequestDto) {
+        Book model = BookMapper.INSTANCE.toModel(bookRequestDto);
```

Comment:
```suggestion
        Book model = bookMapper.toModel(bookRequestDto);
```
Create class-level variable (similar to BookRepository)